src/Provider/BaseCategory.php

```diff
@@ -123,73 +104,4 @@ final public function getSystemInfo(): SystemInfo
 
         return $this->systemInfo;
     }
-
-    /**
-     * Get a guzzle handler stack which logs requests/responses if provider is
-     * an instance of LogsDebugData and $debugLog === true. Requests and responses
-     * will also be stored in $this->guzzleHistory.
-     */
-    protected function getGuzzleHandlerStack(bool $debugLog = false): HandlerStack
```

Comment:
We need to keep this method for PSR-3 logging middleware as it's widely used

Reply:
This is now updated. Static analysis will be fixed again when #60 is merged